blt/urls.py

```diff
@@ -513,6 +517,10 @@
     path("fetch-current-bid/", fetch_current_bid, name="fetch_current_bid"),
     path("Submitpr/", submit_pr, name="submit_pr"),
     path("weekly-report/", weekly_report, name="weekly_report"),
+    path("add-suggestion/", add_suggestions, name="add_suggestions"),
+    path("view-suggestion/", view_suggestions, name="view_suggestions"),
+    path("vote-suggestion/", vote_suggestions, name="vote_suggestions"),
+    path("set-vote/", set_vote, name="set_vote"),
```

Comment:
Could you group them under `/suggestion/` prefix?

Reply:
Sure!